index.md

```diff
@@ -7,6 +7,13 @@ layout: default
 OpenAstronomy is a collaboration between open source astronomy and astrophysics
 projects to share resources, ideas, and to improve code.
 
+## Principles of OpenAstronomy
+
+The core principles of OpenAstronomy include:
+
+1. Open Source: OA projects are committed to their software being Open Source. This promotes accessibility, reliability, and scientific reproducibility.
+2. Open Development: OA projects support the principle of "open development", a model where the software is both *by* and *for* the community.  More concretely, following open development means making code and project-level decisions mainly in publicly-visible spaces. These projects are also open to input from their public user community for these same decisions.
+3. Open Science: OA projects support open and accessible data and reproducible science both through good data management practices and effective open software.
```

Comment:
how far should this go? a pipeline/part of a pipeline, or a generic but highly reusable tool primarily developed for a survey with proprietary data period would fail here?

Reply:
I'm also in favor of leaving data out of the requirements conversation. I think that might touch a nerve that triggers greedy skepticism.

Reply:
The software can "support open and accessible" and be detached from data which has proprietary periods etc. I do agree that this is a fine and more controversial line than the software.